graphql_compiler/compiler/metadata.py

```diff
@@ -161,6 +172,24 @@ def get_location_info(self, location):
                                  u'{}'.format(location))
         return location_info
 
+    @property
+    def outputs(self):
+        for output_name, output_info in six.iteritems(self._outputs):
+            yield output_name, output_info
```

Comment:
why not just `return six.iteritems(self._outputs)`?

Reply:
Hmm I'm not sure if we have a huge reason for one or the other? This is consistent with the other properties.

Reply:
I don't think we have a great reason. It's probably fine either way, we can do some cleanup in a future PR.